src/lib.rs

```diff
@@ -165,14 +165,14 @@ mod tests {
 
             libc_print::libc_println!("{:03}: {:04} {:0.2}", i, count, dif);
 
-            // this is pretty arbitrary, but if we tweak the algorithm
+            // this is pretty arbitrary, but if we tweak the algorithm,
             // and it breaks the tests, at least we'll have to
-            // conciously change this:
+            // consciously change this:
             if do_assert {
                 if i >= 1 && i <= 191 {
                     assert!(dif > 0.5);
                 } else if i >= 194 && i <= 244 {
-                    assert!(dif > 0.07);
+                    assert!(dif > 0.06);
```

Comment:
Minor behavior change, but given the comment above I'm really not to bothered by this. It's just about the distribution of character selection I think

Reply:
Did the tests not pass without that change?

Reply:
They did not no, they failed before I made this change